fdbserver/Status.actor.cpp

```diff
@@ -1038,6 +1038,37 @@ ACTOR static Future<JsonBuilderObject> latencyProbeFetcher(Database cx, JsonBuil
 	return statusObj;
 }
 
+ACTOR static Future<Void> consistencyCheckStatusFetcher(Database cx, JsonBuilderArray *messages, std::set<std::string> *incomplete_reasons, bool isAvailable) {
+	if(isAvailable) {
```

Comment:
If you are going to revise the code, I think it will be better to have `if (!isAvailable) return Void();`. This can help reduce the indentation of the following `try` loop

Reply:
Ok.